### PR TITLE
[hercrentals.com] New ruleset

### PR DESCRIPTION
**Pull request by youdly**

### CODE REVIEW

File: src/chrome/content/rules/Hercrentals.com.xml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+<!--
+	Non-functional subdomains:
+		- airwatch	(expired cert)
+		- autodiscover	(connection refused)
+		- css		(SSL handshake failed)
+		- g		(redirects to invalid www.g)
```
Comment (J0WI) [inline src/chrome/content/rules/Hercrentals.com.xml]: WFM

Comment (youdly) [reply src/chrome/content/rules/Hercrentals.com.xml]: it still redirects to `www.g` which refuse connection. connected via US location.

Comment (J0WI) [reply src/chrome/content/rules/Hercrentals.com.xml]: It works for me in browser, but not with `curl`.
The DNS resolves to multiple endpoints.

Comment (youdly) [reply src/chrome/content/rules/Hercrentals.com.xml]: I see. I added a test url. should I redirect `www.g` (which is not resolvable) to `g`?

Comment (youdly) [reply src/chrome/content/rules/Hercrentals.com.xml]: Redirect doesn't help. Removed `g`.

**Pull request merged**